buildpacks/python/python/build.go

```diff
@@ -81,7 +84,7 @@ func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 		invokerDepCacheLayer,
 		b.CommandRunner,
 		WithValidationLogger(b.Logger),
-		WithValidationFunctionClass(functionClass, isOverride),
+		WithValidationFunctionClass(moduleName, functionName),
```

Comment:
Due to the refactor, the `Function` layer contributer is highly simplified. As a result, it no longer needs to care about whether the function class was set explicitly (i.e. overrides default).